openapi.yaml

```diff
@@ -707,6 +707,14 @@ components:
           $ref: '#/components/schemas/Url'
         license:
           $ref: '#/components/schemas/License'
+        links:
+          type: array
+          items:
+            type: object
+            properties:
+              url:
+                $ref: '#/components/schemas/Url'
+
```

Comment:
@wellemut and @navid-kalaei  do you need a `title` property?

Reply:
Yes optionally, we want to hide the URL behind the title.
this is especially important if we have no standard-URL like facebook or whatsapp and a very long URL.

Reply:
And I just hear the need from Renn Süd to have an optoinal `Tooltip`. When you have a mouse over there should be a Text "Im Werkzeugkasten des Wandels gibt es viele weitere spannende Projekte"
Could you do this tooo?

Reply:
So we would have
- `URL`
- `Title`
- `Tooltip`